paddle.js

```diff
@@ -0,0 +1,38 @@
+class Paddle {
+  constructor(ctx, width, height, x, y, speed, color) {
+    this.ctx = ctx;
+    this.width = width;
+    this.height = height;
+    this.x = x;
+    this.y = y;
+    this.speed = speed;
+    this.color = color;
+    this.direction = 0;
+  }
+
+  move() {
+    if (this.left === 0) {
+      this.setDirection(1);
```

Comment:
why are you setting direction for a paddle hitting the boundaries of a canvas? it should just not move